src/components/new-request/business-fetch.vue

```diff
@@ -61,7 +61,7 @@ export default class BusinessFetch extends Vue {
   private validate (): boolean {
     if (!this.searchField) {
       this.errorMessages = ['Required field']
-    } else if (!/^(A|BC|C|CP|FM)\d{7}$/.test(this.searchField)) {
+    } else if (!/^(A|BC|C|CP|FM|XCP|CCC|CUL)( |)\d{7}$/i.test(this.searchField)) {
```

Comment:
I think it would be better to trim searchField than to allow a trailing space.

I believe the field is already bound like this: `v-model.trim="searchField"` but it may still require an explicit trim() before validating it. (Vuetify has always had this behaviour.)

Reply:
Please test lower-case identifiers with the "COLIN search" (the search that is used when not logged in). Eg, try "cp0000840".